### PR TITLE
feat: Close channels after three days of being open

### PR DESCRIPTION
**Pull request by Brandon-Huu**
Untested lmao

### CODE REVIEW

File: plugins/clopen.py [hunk 1]
```diff
@@ -215,6 +215,8 @@ class Channel:
     extension: Mapped[int] = mapped_column(BigInteger, nullable=False)
     # When to transition to the respective next state
     expiry: Mapped[Optional[datetime]] = mapped_column(TIMESTAMP)
+    creatation_date: Mapped[Optional[datetime]] = mapped_column(TIMESTAMP)
```
Comment (rattus-rattus-rattus) [inline plugins/clopen.py]: There is a typo - `creatation_date`?

File: plugins/clopen.py [hunk 2]
```diff
@@ -215,6 +215,8 @@ class Channel:
...
+
```
Comment (rattus-rattus-rattus) [inline plugins/clopen.py:219]: It seems you didn't run Black. Please refer to [CONTRIBUTING.md](https://github.com/discord-math/bot/blob/main/CONTRIBUTING.md) if you want to contribute to the repo.

File: plugins/clopen.py [hunk 3]
```diff
@@ -265,6 +267,8 @@ async def scheduler_task() -> None:
             for channel in config.channels:
                 async with channel_locks[channel.id]:
                     if channel.state == ChannelState.USED and channel.expiry is not None:
+                        if chanell.creatation_date is not None and ( datetime.utcnow() - chanell.creatation_date).total_seconds > 259200: #three days
+                            await close(session, chanell, "Closed due to exceeding three hours")
```
Comment (rattus-rattus-rattus) [inline plugins/clopen.py]: More typos.

**Review by rattus-rattus-rattus — CHANGES_REQUESTED** [review]
Please refer to [CONTRIBUTING.md](https://github.com/discord-math/bot/blob/main/CONTRIBUTING.md) if you want to contribute to the repo. At a bare minimum, you need to fix your typos, run the black formatter, and actually test your code.

File: plugins/clopen.py (hunk 3 above)
Comment (Brandon-Huu) [reply plugins/clopen.py]: Fixed this earlier

**Comment by Brandon-Huu** [comment]
I'll probably wont finish this. Too busy with school.

**Pull request closed without merging**